app/api/auth/route.ts

```diff
@@ -0,0 +1,56 @@
+import { auth } from 'firebase-admin';
+import { cookies } from 'next/dist/client/components/headers';
+import { NextRequest, NextResponse } from 'next/server';
+
+import { customInitializeApp } from '@/firebase/firebaseAdminConfig';
+
+/* Initi Firebase Admin App once the server is called */
+customInitializeApp();
+
+const EXPIRATION_TIME = 60 * 60 * 24 * 5 * 1000;
```

Comment:
Could you add a comment here showing the final value after the math operation? I mean for it to be easier to read, as in `// Total amount: X days` or `// Total amount: X hours`, not in milliseconds

Reply:
sure